Makefile

```diff
@@ -93,15 +94,15 @@ _episodes/%.md: _episodes_rmd/%.Rmd
 
 ## lesson-check     : validate lesson Markdown.
 lesson-check : lesson-fixme
-	@bin/lesson_check.py -s . -p ${PARSER} -r _includes/links.md
+	${PYTHON} bin/lesson_check.py -s . -p ${PARSER} -r _includes/links.md
 
 ## lesson-check-all : validate lesson Markdown, checking line lengths and trailing whitespace.
 lesson-check-all :
-	@bin/lesson_check.py -s . -p ${PARSER} -r _includes/links.md -l -w --permissive
+	${PYTHON} bin/lesson_check.py -s . -p ${PARSER} -r _includes/links.md -l -w --permissive
 
 ## unittest         : run unit tests on checking tools.
 unittest :
-	@bin/test_lesson_check.py
+	${PYTHON} bin/test_lesson_check.py
```

Comment:
As is, this change does not aid Win 10 compatibility. Let's improve Makefile in a separate PR.
(Also, let's keep those `@`)

Reply:
See my general comment on the PR.

Reply:
(Put the @s back in)